opennms-config-jaxb/src/main/java/org/opennms/netmgt/config/snmp/Configuration.java

```diff
@@ -942,6 +957,13 @@ public boolean equals(final Object obj) {
         } else if (!m_writeCommunity.equals(other.m_writeCommunity)) {
             return false;
         }
+        if (m_ttl == null) {
+            if (other.m_ttl != null) {
+                return false;
+            }
+        } else if (!m_ttl.equals(other.m_ttl)) {
+            return false;
+        }
```

Comment:
If `m_encrypted` is included in the hashcode above, shouldn't it be part of the `equals()` too?

Reply:
yeah, it doesn't need to be part of hascode as well. Don't need this for comparison